client/src/Main.elm

```diff
@@ -410,13 +389,17 @@ view model =
         [ Element.layout
             [ Element.height Element.fill
             , inFront <| Notify.view CloseNotification model.notifications
+            , inFront <| navLinks
```

Comment:
Why is this necessary?

Reply:
Because otherwise the navbar  isn't sticky anymore.

Reply:
Ah ok :+1: